liferay-tango/liferay-portal
# LPS-151362 Add an Alert in Experiences dropdown when segmentation is disabled

Motivation
=======
**As a** Marketer, 
**I want to** be warned in context when Segmentation is disabled, 
**so that** I am aware that my Segments, Experiences and Collections will not function properly without this configuration enabled.

[Link to story or ticket](https://issues.liferay.com/browse/LPS-151362)

Proposed Solution
========
Warning message added to the Experiences Dropdown selector when editing a page:

"**Experiences can not be displayed because segmentation is disabled.** To enable Segmentation go to Instance Settings > Segments > Segments Service."


Steps to Verify:
----------------
1. Disable Segmentation: Go to _Instance Settings > Segments > Segments Service_.
2.  Go to main page and click the edit icon
3.  Deploy the Experiences dropdown
4. The corresponding message should be visible:
- If segmentation is enabled:  **Warning** Changes to experiences are applied immediately
- If segmentation is disabled: **Experiences can not be displayed because segmentation is disabled.** To enable Segmentation go to Instance Settings > Segments > Segments Service.

Change summary (optional):
---------------
- LPS-151362 Added segmentation disabled warning message to the experience selector
- LPS-151362 buildLang

Screenshots:
<img width="1654" alt="Experiences" src="https://user-images.githubusercontent.com/85171101/170242488-e6280530-4cc9-44cc-bbd7-e3448cc232cc.png">
